src/crypto/CHIPCryptoPALPSA.cpp

```diff
@@ -106,113 +106,118 @@ bool isValidTag(const uint8_t * tag, size_t tag_length)
 
 } // namespace
 
+AesKey::AesKey()
+{
+    Get<psa_key_id_t>() = 0;
+}
+
+AesKey::~AesKey()
+{
+    psa_destroy_key(Get<psa_key_id_t>());
+}
+
+CHIP_ERROR AesKey::Initialize(const AesKeyByteArray & rawKey)
+{
+    const psa_algorithm_t algorithm = PSA_ALG_AEAD_WITH_AT_LEAST_THIS_LENGTH_TAG(PSA_ALG_CCM, 8);
+    CHIP_ERROR error                = CHIP_NO_ERROR;
+    psa_status_t status             = PSA_SUCCESS;
+    psa_key_id_t keyId              = 0;
```

Comment:
0 is fine anyway..
```suggestion
    psa_key_id_t keyId              = PSA_KEY_ID_NULL;
```

Reply:
Earlier versions of mbedTLS don't have this constant. That's why I use 0 for better portability.

Reply:
You could introduce a locally-scoped constant in anonymous namespace that is set to PSA_KEY_ID_NULL if present, so as to not assume it's always 0.